app/controllers/reports_controller.rb

```diff
@@ -35,6 +35,7 @@ def edit
   # POST /reports.json
   def create
     @report = Report.new(report_params)
+    @report.background = Background.find(1) unless @report.background
```

Comment:
This is a bit dangerous. 
You can implement a function called
Background.default
and where you specify the name for it and find_by. 
harcoding find(1) is not good.